src/main/java/no/nav/fo/veilarb/dokument/config/ClientConfig.java

```diff
@@ -28,12 +30,19 @@ public EnhetClient enhetClient(EnvironmentProperties properties) {
     }
 
     @Bean
-    public SakClient sakClient(EnvironmentProperties properties) {
-        return new SakClientImpl(RestClient.baseClient(), clusterUrlForApplication("sak"));
+    public SakClient sakClient() {
+        return new SakClientImpl(RestClient.baseClient(), naisPreprodOrNaisAdeoIngress("sak", false));
     }
 
     @Bean
-    public VeilederClient veilederClient(EnvironmentProperties properties) {
-        return new VeilederClientImpl(RestClient.baseClient(), clusterUrlForApplication("veilarbveileder", true));
+    public VeilederClient veilederClient() {
+        return new VeilederClientImpl(RestClient.baseClient(), naisPreprodOrNaisAdeoIngress("veilarbveileder", true));
     }
+
+    private static String naisPreprodOrNaisAdeoIngress(String appName, boolean withAppContextPath) {
+        return  EnvironmentUtils.isDevelopment().orElse(false)
```

Comment:
Ville kanskje snudd denne og brukt `isProduction`

Reply:
Litt uenig. Nå så sjekker vi om vi ligger i List.of("dev-fss", "dev-sbs", "dev-gcp") og hvis ikke så er vi i prod. Hvis vi gjør det motsatt så er det større sjanse for at sjekken mot et av prod clusterene feiler og vi faller tilbake til å bruke dev